### DISCUSSION TITLE
Is there a way to run `git lfs pull` to only fetch one big object?

### DISCUSSION THREAD

**Discussion by flexwang2** [git-lfs/git-lfs#5131] [Q&A]
No matter I run `git lfs pull` or `git lfs pull large.bin`, I end up downloading all the large files. Is there a way to download only one of them by specify the path?

**Comment by chrisd8088** [comment]
Yes, use the `-I` or `--include=` options, as documented in the manual [page](https://github.com/git-lfs/git-lfs/blob/880751d2498f94907bf6a37d4d5da7c815937f4d/docs/man/git-lfs-pull.adoc#options), which is also available using `git lfs help pull`.  (All of the `git lfs` commands should have documentation you can access that way.)  So you'd want something like `git lfs pull -I large.bin`.
